src/locales/locale-en.json

```diff
@@ -94,7 +94,18 @@
     "NONE": "No upcoming activities",
     "NONE_DUE_TO_FILTER": "No activities match your filters",
     "NONE_HINT": "You can create a new activity on the place page.",
-    "STORE_NONE_HINT": "You can create a new activity on the 'Manage Activities' page."
+    "STORE_NONE_HINT": "You can create a new activity on the 'Manage Activities' page.",
+    "ICS_DIALOG": {
+      "TITLE": "Add to calendar",
+      "INTRO": "Add these activities to your calendar by using one of the following options:",
+      "DOWNLOAD_TITLE": "Download",
+      "DOWNLOAD_TEXT": "Download the ICS-file to manually import it into your calendar:",
+      "DOWNLOAD_BUTTON_LABEL": "Download ICS",
+      "SUBSCRIBE_TITLE": "Subscribe",
+      "SUBSCRIBE_TEXT": "Use this URL to subscribe as an ICS calendar:",
+      "SUBSCRIBE_URL_COPIED_TOAST": "The URL was successfully copied to your clipboard",
```

Comment:
Now I just added the toast string to my PR #2349 so maybe we should not have it under ICS_DIALOG?

I put it on a top level named URL_COPIED_TOAST together with URL_CLICK_TO_COPY

Reply:
changed accordingly ;)